src/west/commands/project.py

```diff
@@ -32,22 +35,71 @@ def __init__(self):
             _wrap('''
             List projects.
 
-            Prints the path to the manifest file and lists all projects along
-            with their clone paths and manifest revisions. Also includes
-            information on which projects are currently cloned.
-            '''))
+            Individual projects can be specified by name.
+
+            By default, lists all project names in the manifest, along with
+            each project's path, revision, URL, and whether it has been cloned.
+
+            The west and manifest repositories in the top-level west directory
+            are not included by default. Use --all or the special project
+            names "west" and "manifest" to include them.'''))
 
     def do_add_parser(self, parser_adder):
-        return _add_parser(parser_adder, self, _project_list_arg)
+        default_fmt = '{name:14} {path:18} {revision:13} {url} {cloned}'
```

Comment:
Maybe roll this into the parameter below, like `default='{name:14} {path:18} {revision:13} {url} {cloned}'`. That makes it clear that it's used in a single spot and never updated.

Reply:
I am aware of your antipathy towards single-use variables. The one below is leftovers from rebasing.

This one is used in two places, however:

1. as the default kwarg
2. in the epilog

Reply:
Ah, sorry, search fail.